packages/mailer/handlers/resend/src/index.ts

```diff
@@ -9,7 +8,13 @@ import type {
 import { AbstractMailHandler } from '@redwoodjs/mailer-core'
 
 export type ResendMailHandlerOptions = {
-  tags?: Tag[]
+  // Note: Resend SDK no longer exports the type Tag but it's simple enough to copy
+  // out here. It just makes us a little more susceptible to getting out of sync if
+  // the Resend SDK changes.
+  tags?: {
+    name: string
+    value: string
+  }[]
```

Comment:
I'm not thrilled with this, I'd rather continued to use an exported type from the package, but such is life.